guides/common/modules/proc_booting-virtual-machines.adoc

```diff
@@ -48,14 +51,3 @@ If you want to change the default values in the template, clone the template and
 . Set *PXE Loader* to *iPXE Embedded*.
 . Select the *Templates* tab.
 . From the *iPXE template* list, select *Review* to verify that the *Kickstart default iPXE* template is the correct template.
-. On {ProjectServer}, run:
```

Comment:
I see it's removed, so does it mean it's not needed anymore?

Reply:
It works out of the box for content proxies, but not `{ProjectServer}`. You also still need to ensure `--foreman-trusted-proxies` is set correctly for any content proxy.

Reply:
Trusted proxies are moved above into a Prerequisite (the snippet).

Reply:
@ekohl Does the user need to set the HTTP URL only when they boot hosts via ProjectServer or even when they use SmartProxy to boot the hosts?

Reply:
For ProjectServer we don't have code to set the HTTP URL, so that's needed *if* you want to use ProjectServer to provision. Technically there's no reason to if you already have a SmartProxy doing that.

For SmartProxy I think the bootstrap part is needed, so `{foreman-installer} --foreman-proxy-dhcp-ipxe-bootstrap true` is the correct command for content proxies.

It's good that you mention the Prequisite. I'd suggest to move it to `Configuring {SmartProxy} for Host Registration and Provisioning`. You already call the installer there and that's the right place. It also

Reply:
@ekohl Does it work the same in Foreman 3.1?